packages/app/components/Queue/TA/TAAddStudent.tsx

```diff
@@ -197,6 +238,35 @@ export function AddStudentsModal({
             ) : (
               <p>There are no students or all students are in queue</p>
             )}
+            {selectOptions.length > 0 ? (
+              <>
+                <br />
+                <Form onFinish={handleSubmit}>
+                  {selectOptions.map((option, index) => (
+                    <div key={index}>
+                      <strong>
+                        <p>{option.value}</p>
+                      </strong>
+                      <Form.Item>
+                        <input
+                          placeholder={`Enter ${option.value}'s question`}
+                          style={{ width: "100%", height: "40px" }}
+                          onChange={(e) => handleChange(e, option.id)}
```

Comment:
If we only need changed value, why not pass only value e.g. `e.target.value` instead of the whole object?